cmd/docker.go

```diff
@@ -66,7 +72,23 @@ func execDockerRun(image string, command []string) {
 		}
 	}
 
-	args = append(args, "--volume", workDir+":/app:delegated", image)
+	args = append(args, "--volume", workDir+":/app:delegated")
+
+	if len(dockerFlags.Volumes) > 0 {
+		for _, volume := range dockerFlags.Volumes {
+			args = append(args, "--volume", volume)
+		}
+	}
+
+	if dockerFlags.Expose != "" {
+		args = append(args, "--expose", dockerFlags.Expose)
+	}
+
+	if dockerFlags.Publish != "" {
+		args = append(args, "--publish", dockerFlags.Publish)
```

Comment:
For `--publish` (short `-p`) I guess we should also make it an array like Volumes and Envs... there are common cases that we want to export two ports from a `kool docker` (i.e when serving a Node application and we want to connect DevTools to it).